extensions/settings/_oscar.py

```
@@ -135,11 +135,18 @@
 
 
 # PAYMENT PROCESSING
-PAYMENT_PROCESSOR_CLASS = 'extensions.payment.processors.Cybersource'
+PAYMENT_PROCESSORS = (
+    'extensions.payment.processors.Cybersource',
+)
 
-CYBERSOURCE_PROFILE_ID = None
-CYBERSOURCE_ACCESS_KEY = None
-CYBERSOURCE_SECRET_KEY = None
+PAYMENT_PROCESSOR_CONFIG = {
+    'cybersource': {
+        'profile_id': 'set-me-please',
+        'access_key': 'set-me-please',
+        'secret_key': 'set-me-please',
+        'pay_endpoint': 'fake-secret-key',
```

Minor: It might be good to change the default to a sample URL; this seems to suggest that `pay_endpoint` is a secret key.